src/main/kotlin/lotto/model/WinNumber.kt

```diff
@@ -37,7 +28,7 @@ class WinNumber private constructor(
             require(!lastWinNumber.isNullOrBlank())
             require(bonusNumber != null)
 
-            return WinNumber(parsingTextToLotto(lastWinNumber), LottoNumber(bonusNumber))
+            return WinNumber(Lotto.parsingTextToLotto(lastWinNumber), LottoNumber(bonusNumber))
```

Comment:
```suggestion
            return WinNumber(
                _lastWinNumber = Lotto.parsingTextToLotto(lastWinNumber),
                _bonusNumber = LottoNumber(bonusNumber)
            )
```
함수의 파라미터가 무엇인지 잘 모를 경우에는 argument name을 활용해주시면 좋습니다